### ISSUE TITLE
SAIGE running error

### ISSUE THREAD

**Issue by huanghe0220** [weizhouUMICH/SAIGE#344]
Hi @weizhou0 @weizhouUMICH @matuskosut and others: 
I downloaded the saige with conda following this instruction[https://github.com/weizhouUMICH/SAIGE/issues/272](url)
`conda create -n saige -c conda-forge -c bioconda r-base=3.6.1 r-saige=0.38.8.8 savvy=1.3.0 r-optparse=1.6.2=r36h6115d3f_1`
And succeeded in these three steps
`step1_fitNULLGLMM.R --help
step2_SPAtests.R --help
createSparseGRM.R --help`

However, when I ran step 1(`Rscript step1_fitNULLGLMM.R`) with my own dataset, I got a bug like this:
`Error in terms.formula(formula, data = data) : invalid model formula in ExtractVars`

when I ran step 1 with the example dataset, I got
 `Error in fitNULLGLMM(plinkFile = opt$plinkFile, phenoFile = opt$phenoFile,  :
  unused arguments (IsOverwriteVarianceRatioFile = opt$IsOverwriteVarianceRatioFile, includeNonautoMarkersforVarRatio = opt$includeNonautoMarkersforVarRatio, sexCol = opt$sexCol, FemaleCode = opt$FemaleCode, FemaleOnly = opt$FemaleOnly, MaleCode = opt$MaleCode, MaleOnly = opt$MaleOnly, noEstFixedEff = opt$noEstFixedEff, useSparseGRMtoFitNULL = opt$useSparseGRMtoFitNULL)`

If I annotate those argument then ran step 2(`Rscript step2_SPAtests.R`) with the example dataset, I bumped into
 `Error in getopt_options(object, args) : Error in getopt(spec = spec, opt = args) : long flag "dosageFile" is invalid`

I don't know why this happened. Do you have any advise? Looking forward to your reply.

Best,
Huang He

**Comment by matuskosut** [comment]
Hi @huanghe0220 , can you try to recreate environment in simple way
```
conda create -n saige -c conda-forge -c bioconda -y r-base=3.6.3 r-saige=0.35.8.8
conda activate saige
```
What is the output of this command?
```
which step2_SPAtests.R
```
When you run help do you get correct version of saige in the output?
```
step2_SPAtests.R --help
```
ps: step1 says unused arguments, good to read older issues or wiki

**Comment by weizhouUMICH** [comment]
Sorry fo the late reply! We have just released a new version 1.0.0. It has substantial computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests and clearer log output. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
